/*
Copyright 2020 The Magma Authors.

This source code is licensed under the BSD-style license found in the
LICENSE file in the root directory of this source tree.

Unless required by applicable law or agreed to in writing, software
distributed under the License is distributed on an "AS IS" BASIS,
WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
See the License for the specific language governing permissions and
limitations under the License.
*/
syntax = "proto3";

package magma.lte;
option go_package = "magma/lte/cloud/go/protos";

message DeleteSubscriberResponse {
}

message DeleteSubscriberRequest {
    // a list of imsi of the subscriber to delete
    repeated string imsi_list = 1;
}

service S6aService {
    // Delete a subscriber from mme. A grpc service in mme, client req initiated from subscriberdb.
    rpc DeleteSubscriber (DeleteSubscriberRequest) returns (DeleteSubscriberResponse) {}
}
